src/lib/components/TagTypeSelection.svelte

```
@@ -0,0 +1,45 @@
+	>
+{/snippet}
+
+<Select.Root type="multiple" bind:value={tagtypes} name="tagtypes" {...rest}>
+	<Select.Trigger>
+		{#if tagtypes?.length === 0}
+			{selectedTagtypes}
+		{:else}
+			<span class="truncate text-start">
+				{#each selectedTagtypes as tagtype}
+					{@render myBadge(tagtype)}
+				{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)